src/themes/default/components/core/blocks/MyAccount/MyOrder.vue

```diff
@@ -139,13 +145,33 @@
 <script>
 import MyOrder from '@vue-storefront/core/compatibility/components/blocks/MyAccount/MyOrder'
 import ReturnIcon from 'theme/components/core/blocks/Header/ReturnIcon'
+import { getThumbnailPath } from '@vue-storefront/core/helpers'
+import { mapActions } from 'vuex'
 
 export default {
   mixins: [MyOrder],
   components: {
     ReturnIcon
+  },
+  data () {
+    return {
+      itemThumbnail: []
+    }
+  },
+  methods: {
+    ...mapActions({
+      getProduct: 'product/single'
+    }),
+    getProductThumbnail (productSku) {
+      if (this.itemThumbnail[productSku] === undefined || this.itemThumbnail[productSku] === null) {
```

Comment:
it is the same as this, but more readable
```suggestion
      if (!this.itemThumbnail[productSku]) {
```